rivasvict/wedding-app
# Production ready

Hi Team, This PR faces #24 issue about making the code production-ready. Ugglified. Please remember to `npm install`

To test:
For production:
`npm run serve-production`
(check that this is ugglified)
and for development:
`npm run serve-dev`
